src/qute/languageServer/client.ts

```diff
@@ -79,6 +79,7 @@ export function connectToQuteLS(context: ExtensionContext) {
       if (!hasShownQuteValidationPopUp(context)) {
         await showQuteValidationPopUp(context);
       }
+      await setQuteValidationEnabledContext();
```

Comment:
Need to add this to a listener so there is a check each time the button state changes (based on `quteValidationEnabled`)